### ISSUE TITLE
Memory scanner and game list data

### ISSUE THREAD

**Issue by mataha** [blacha/diablo2#276]
Can the memory scanner (or rather: the technique) be used to reliably pull game list data from within D2R?

**Comment by blacha** [comment]
Game list? as in the list of online games you can join? If so no.

The memory reader is focused on the original diablo2 game structures, which gives you attrs like HP/Mana/Experience, Items, nearby mosnters etc..

**Comment by mataha** [comment]
I know that, in the current state, it is possible to pull player/map data only.
I might've beaten around the bush, so let me rephrase my question: could I modify that code to get a list of online lobbies?

**Comment by blacha** [comment]
They lobby list is pretty easy to find in memory (I think) I am not sure how often it is updated though, it may only refresh if you are viewing the page.

It might be a option to load the game in a memory reader and just search for a known game name from the list, it should be pretty quick to see the games lists.
